lib/internal/errors.js

```diff
@@ -1403,6 +1403,7 @@ E('ERR_UNKNOWN_FILE_EXTENSION',
   TypeError);
 E('ERR_UNKNOWN_MODULE_FORMAT', 'Unknown module format: %s', RangeError);
 E('ERR_UNKNOWN_SIGNAL', 'Unknown signal: %s', TypeError);
+E('ERR_UNSUPPORTED_DIR_IMPORT', "Cannot import '%s' from %s.", Error);
```

Comment:
We can perhaps add a little more context to the message eg - `Directory import '%s' is not supported resolving ES modules, imported from %s`.

Note also I usually try to avoid a `.` after a full path as vscode doesn't highlight these as links properly in the terminal.